msal-dotnet-articles/advanced/backup-authentication-system.md

```diff
@@ -1,26 +1,28 @@
 ---
 title: Backup authentication system
```

Comment:
Maybe this page is a good candidate to move into a more general resiliency page, like High Availability, as a section.

Reply:
Indeed, but I will defer that to another pull request rather than this one.